bidrag-domene/src/main/kotlin/no/nav/bidrag/domene/enums/inntekt/Inntektsrapportering.kt

```diff
@@ -185,6 +185,24 @@ enum class Inntektsrapportering(
             ),
     ),
 
+    BARNETILLEGG(
+        hentesAutomatisk = true,
+        kanLeggesInnManuelt = true,
+        inneholderInntektstypeListe =
+            listOf(
+                Inntektstype.BARNETILLEGG_PENSJON,
+                Inntektstype.BARNETILLEGG_UFØRETRYGD,
+                Inntektstype.BARNETILLEGG_DAGPENGER,
+                Inntektstype.BARNETILLEGG_KVALIFISERINGSSTØNAD,
+                Inntektstype.BARNETILLEGG_AAP,
+                Inntektstype.BARNETILLEGG_DNB,
+                Inntektstype.BARNETILLEGG_NORDEA,
+                Inntektstype.BARNETILLEGG_STOREBRAND,
+                Inntektstype.BARNETILLEGG_KLP,
+                Inntektstype.BARNETILLEGG_SPK,
+            ),
+    ),
+
     // Manuelt registrerte rapporteringer
 
```

Comment:
Er det forskjell på barnetillegg pensjon og barnetillegg uføretrygd? Synes jeg har hørt at de refereres om hverandre (men er vel egentlig mer et spm til Tonje og Christian)

Reply:
Det finnes barnetillegg for både alderspensjon og uføretrygd så vidt jeg vet, men vi får ikke angitt hvilken det gjelder i responsen fra Pesys.